bitnami/contour-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,173 @@
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.podAffinityPreset "component" "contour-operator" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.podAntiAffinityPreset "component" "contour-operator" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.nodeAffinityPreset.type "key" .Values.nodeAffinityPreset.key "values" .Values.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      initContainers:
+        {{- if .Values.initContainers }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      containers:
+        {{- if and .Values.metrics.enabled .Values.metrics.proxy.enabled }}
+        - name: kube-rbac-proxy
+          image: {{ template "contour-operator.proxy.image" . }}
+          imagePullPolicy: {{ .Values.metrics.proxy.image.pullPolicy | quote }}
+          args:
+            - --secure-listen-address=0.0.0.0:{{ .Values.metrics.proxy.containerPort }}
+            - --upstream=http://127.0.0.1:{{ .Values.containerPort }}/
+            - --logtostderr=true
+            - --v=10
+          ports:
+            - containerPort: {{ .Values.metrics.proxy.containerPort }}
+              name: metrics-proxy
+          {{- if .Values.metrics.proxy.resources }}
+          resources: {{- toYaml .Values.metrics.proxy.resources | nindent 12 }}
+          {{- end }}
+        {{- end }}
+        - name: contour-operator
+          image: {{ template "contour-operator.operator.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
+          {{- else }}
+          command:
+            - /contour-operator
```

Comment:
I have seen that the entrypoint for the image is already `contour-operator` https://github.com/bitnami/bitnami-docker-contour-operator/blob/20d19a95296cc93a22fdd35e8f8c52edb8a670fc/1/scratch/Dockerfile#L9  . This should not be necessary.